build/scripts/entrypoint.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash -x
+
+# start httpd
+if [[ -x /usr/sbin/httpd ]]; then /usr/sbin/httpd -D FOREGROUND
+#elif [[ -x /usr/bin/run-httpd ]]; then /usr/bin/run-httpd
+fi
```

Comment:
Sorry, I merged #110 before looking at this PR. The master branch of repo now contains `build/dockerfiles/entrypoint.sh`, so we should use/modify that instead.